src/applications/check-in/components/layout/Footer.jsx

```diff
@@ -3,9 +3,9 @@ import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
```

Comment:
I moved this file into a new layout folder with the new Wrapper component - a lot of the file changes in this PR are just updating imports